### DISCUSSION TITLE
Quarkus 3.3.0 released - OpenTelemetry improvements, Reactive Messaging Pulsar extension

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#35514] [Quarkus Blog/Website]
# Quarkus 3.3.0 released - OpenTelemetry improvements, Reactive Messaging Pulsar extension

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-3-0-released/

**Comment by adampoplawski** [comment]
Hello
It is interesting that Quarkus provides Reactive Messaging extensions. Are there maybe some plans to add SQS support?
